packages/headless-plugin-ext/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@theia/headless-plugin-ext",
+  "version": "1.44.0",
+  "description": "Theia - Backend-only Plugin Extension",
+  "main": "lib/common/index.js",
```

Comment:
There is no `index` module or ambient module in `src/common`. This looks like maybe a stowaway from `@theia/plugin-ext`

Reply:
This is fixed in commit f100932d6c68cd309a3ba08adcd82531012bb72c.